src/analytics.ts

```diff
@@ -61,6 +60,9 @@ export function createAnalyticsInstance(options?: Options) {
     const setUrl = (href: string) => _growthbook?.setUrl(href)
     const getId = () => _rudderstack?.getUserId() || _rudderstack?.getAnonymousId()
 
+    // for QA testing purposes
+    window.getMyId = () => getId()
+
```

Comment:
do you think we should expose this on production ?

Reply:
actually we should have this exploit for testing purpose and QA needs